packages/jest-circus/src/utils.js

```diff
@@ -44,14 +45,15 @@ export const makeDescribe = (
 
 export const makeTest = (
   fn: ?TestFn,
-  mode: TestMode,
+  _mode: TestMode,
```

Comment:
underscore may indicate that it's optional param. Can we change it back to `mode` and rename the local variable?